File: patterns/Pattern11.java
import java.util.*;

class Pattern11{

	public static void main(String args[]){
		Scanner sc= new Scanner(System.in);

		int n;
		System.out.print("Enter a range: ");
		n= sc.nextInt();
		
		for(int i=0; i<=n; i++)
		{
			for(int j=0; j<=n-i; j++)
			{
				System.out.print(" ");
			}
			for(int k=0; k<=i; k++)
			{
				System.out.print("*");
			}
			System.out.println();
		}
		for(int i=0; i<=n; i++)
		{
			for(int k=0; k<=i; k++)
			{
				System.out.print(" ");
			}
			for(int j=0; j<=n-i; j++)
			{
				System.out.print("*");
			}
			System.out.println();
		}
	}
}

File: pattern printing_java_c++/pattern_11.cpp
#include<iostream>
using namespace std;
int main()
{
	int n;
	cout <<"Enter a range: ";
	cin >>n;
	
	for(int i=0; i<=n; i++)
	{
		for(int j=0; j<=n-i; j++)
		{
			cout <<" ";
		}
		for(int k=0; k<=i; k++)
		{
			cout <<"*";
		}
		cout <<endl;
	}
	for(int i=0; i<=n; i++)
	{
		for(int k=0; k<=i; k++)
		{
			cout <<" ";
		}
		for(int j=0; j<=n-i; j++)
		{
			cout <<"*";
		}
		cout <<endl;
	}
	return 0;
}